Composer/packages/extensions/intellisense/src/components/CompletionElement.tsx

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { CompletionItemKind } from 'monaco-languageclient';
+import { FontIcon } from 'office-ui-fabric-react/lib/Icon';
+import { DirectionalHint, TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
+import React, { CSSProperties } from 'react';
+import { CompletionItem, MarkupContent } from 'vscode-languageserver-types';
+
+type FuseJsMatch = { indices: number[][]; value: string; key: string };
+
+const styles: Record<string, CSSProperties> = {
+  completionElement: {
+    height: '32px',
+    cursor: 'pointer',
+    padding: '0 4px',
+    width: '100%',
+    display: 'flex',
+    alignItems: 'center',
+  },
+  selectedElement: { backgroundColor: '#ddd' },
+  text: { fontSize: '15px' },
+  icon: { marginRight: '5px' },
+};
+
+const renderIcon = (kind: CompletionItemKind | undefined): JSX.Element => {
+  let icon: JSX.Element = <> </>;
+
+  switch (kind) {
+    case CompletionItemKind.Function:
+      icon = <FontIcon iconName="Variable" style={styles.icon} />;
+      break;
+    case CompletionItemKind.Variable:
+      icon = <FontIcon iconName="VariableGroup" style={styles.icon} />;
+      break;
+    case CompletionItemKind.Enum:
+      icon = <FontIcon iconName="BulletedList" style={styles.icon} />;
+      break;
+  }
+
+  return icon;
+};
+
```

Comment:
Nit: You generally can let the return value be inferred.  Sometimes this requires putting the HTML in parens.  Consider either always having it or not - for consistency in this file

Reply:
The other comment addressed this :)